grr/server/grr_response_server/gui/ui/components/flow_details/plugins/osquery_details.ts

```diff
@@ -80,6 +80,24 @@ export class OsqueryDetails extends Plugin {
       map(result => result.stderr),
   );
 
+  readonly exportCsvLink$ = this.flowListEntry$.pipe(
+      map(fle => {
+        const clientId = fle.flow.clientId;
+        const flowId = fle.flow.flowId;
+
+        if (clientId && flowId) {
+          return `/api/clients/${clientId}/flows/${flowId}/osquery-results/CSV`;
+        } else {
+          return null;
+        }
+      }),
+      filter(isNonNull),
+  );
+
+  readonly atLeastOneRowAvailable$ = this.displayTable$.pipe(
+      map(table => table.rows?.length),
```

Comment:
`atLeastOneRowAvailable` sounds like a boolean, but the underlying type seems to be number. Please rename if you'd like to keep the type, or convert to a boolean if you'd like to keep the name.